news/_posts/2023-02-25-release-20230225.md

```diff
@@ -0,0 +1,24 @@
+---
+kind: article
+title: "Release 20230225"
+author: "PunkPun"
+created_at: 2023-02-25 00:00 +0000
+disqus_id: "release-20230225"
+permalink: "/news/release-20230225/"
+---
+
+Today, we have a special occasion on our hands. We have been waiting for this moment for almost two years: a new stable OpenRA [Release 20230225]({{ '/download' | relative_url }}) is out now!
+
+<figure>
+  <video autoplay loop muted>
+    <source src="{{ '/images/20230225-2023.mp4' | relative_url }}" type="video/mp4">
+    <source src="{{ '/images/20230225-2023.webm' | relative_url }}" type="video/webm">
+    <img src="{{ '/images/news/20230225-2023.png' | relative_url }}"/>
+  </video>
+</figure>
+
+It has been a long road to this release. It is filled both with large features, and many important smaller scale polish. For more details see the previous news posts [[one]({{ '/news/playtest-20221203/' | relative_url }}), [two]({{ '/news/playtest-20230110/' | relative_url }}), [three]({{ '/news/playtest-20221223/' | relative_url }}), [four]({{ '/news/playtest-20230218/' | relative_url }})] and, as always, in the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/).
+
+However, there is more! There has been great progress made towards supporting remastered assets. Those who have been following us on [Discord](https://discord.openra.net) may already know, we have a multiplier-compatible HD Tiberian Dawn release! In case you are interested and own a copy of the remasters, you can download it [here](https://github.com/pchote/TiberianDawnHD/releases), but do keep in mind that it is still work-in-progress and does not reflect the final product.
```

Comment:
```suggestion
However, there is more! There has been great progress made towards supporting remastered assets. Those who have been following us on [Discord](https://discord.openra.net) may already know, we have a multiplayer-compatible HD Tiberian Dawn release! In case you are interested and own a copy of the remasters, you can download it [here](https://github.com/pchote/TiberianDawnHD/releases), but do keep in mind that it is still work-in-progress and does not reflect the final product.
```